maflib/schemas/gdc-2.0.0-aliquot-merged.json

```diff
@@ -0,0 +1,111 @@
+{
+	"version" : "gdc-1.0.0",
+	"annotation-spec" : "gdc-2.0.0-aliquot-merged",
+	"extends" : "gdc-1.0.0",
```

Comment:
Shouldn't this technically extend `gdc-1.0.0-aliquot-merged` with the appropriate columns `filtered`?

Reply:
yup, just lazy

Reply:
I stopped being lazy and did the inheritance